lua/apple-music/init.lua

```diff
@@ -328,6 +379,18 @@ M.select_album_telescope = function()
 	}):find()
 end
 
+---Get the name of the current track.
+---@usage require('apple-music').get_current_trackname()
+M.get_current_trackname = function()
```

Comment:
I'd rather this be a local function rather than exported with the module. My reason is because I'd like to add a function to get `current_track_info()` and this will be unnecessary when that is around.

It will also help us if we split up the module a bit more like you suggested in discussion.

Reply:
Okay, that sounds fine.